.github/actions/deploy-happy-stack/action.yml

```diff
@@ -54,6 +54,7 @@ runs:
         TAG: ${{ inputs.tag }}
         ENV: ${{ inputs.env }}
         ENV_FILE: ${{ inputs.env-file }}
+        DOCKER_COMPOSE_PATH: ${{ inputs.docker-compose-path }}
```

Comment:
I think you will also need to add an input at the top of this file: 

https://github.com/chanzuckerberg/github-actions/blob/0974ca9a40d1fc500a675e4d7fff8bd307655ce5/.github/actions/deploy-happy-stack/action.yml#L3

also, to make it a little more clear, can we call it `docker-compose-config-path`

Reply:
Good catch thank you!